source/javascripts/app/_search.js

```diff
@@ -32,18 +33,32 @@
     content = $('.content');
     searchResults = $('.search-results');
 
-    $('#input-search').on('keyup', search);
+    $('#input-search').on('keyup', debounced_search(debounce_delay));
   }
 
+  function debounced_search(wait) {
```

Comment:
Nitpick, but I would uppercase the `debounce_delay` constant, and change the parameter name here to match the same naming style, so like `delay` instead of `wait`